### ISSUE TITLE
Does sourcebans work on a linux server?

### ISSUE THREAD

**Issue by OMGLent** [GameConnect/SourceBans#70]
So I have been trying to get source bans for awhile. Basically hasn't happened today I realized that maybe it was because I'm using an Linux server, Just trying to get some verification if that is the case.

**Comment by Cruckes** [comment]
I've been using Sourcebans on Debian since I started to use SB. Nowdays I use SB 2.0 and works fine for me. Can you specify your problem?

**Comment by OMGLent** [comment]
Well I use ServerGuard which is a build has a built in plugin but I get the following error.

[ERROR] addons/serverguard/lua/plugins/sourcebans/init.lua:16: No Database Connection

error - [C]:-1
GetAllActiveBans - addons/serverguard/lua/plugins/sourcebans/init.lua:16
LOAD_ACTIVE_BANS - addons/serverguard/lua/plugins/sourcebans/init.lua:14
v - addons/serverguard/lua/plugins/sourcebans/init.lua:48
unknown - lua/includes/modules/hook.lua:84

I know I have the right database details in my server and the server comes with MySQLOO by default so I'm not sure what the issue seems to be. I talked to the owner of ServerGuard and hes basically saying that the plugin would unfortunately require a complete recode.

His post
"The Sourcebans plugin is one that has been causing constant errors. We're aware of it's issues but unfortunately the fix is an entire recode. The error you're seeing it displaying that there is no database connection, it could be due to incorrect credentials or even a missing mysql module."

I'm 100% sure the code for the Database is correct, I also had another error on the website side of sourcebans where although I would put in my server details It would receive no query from the server, but if I clicked the connect button I would join without Issue. I assumed this was because My server is Linux however it may be the DDoS Protection provided by my hoster.
